CHANGELOG.md

```diff
@@ -2,9 +2,10 @@
 
 ## master / unreleased
 
+* [CHANGE] Upgraded gogo/protobuf tool chain from 1.2.1 to 1.3.0. #2055
```

Comment:
We keep track only of changes which are visible to the final user. This change doesn't look so, so I would suggest to remove this entry (and so to rollback all `CHANGELOG.md` changes).

Reply:
Why was this marked as resolved?